packages/mobile/__tests__/pages/Login.test.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { render, fireEvent, wait, waitForElement } from '@testing-library/react-native';
+import configureStore from 'redux-mock-store';
+import AsyncStorage from '@react-native-community/async-storage';
+import { showMessage } from 'react-native-flash-message';
+
+import Wrapper from '../../helpers/Wrapper';
+import { handleLoginFacebook } from '../../src/services/authFacebook';
+import Login, { USER_LOGIN_MUTATION } from '~/pages/Login/index';
+import { createApolloStore } from '../../fixtures/login';
+
+const navigation = {
+  dispatch: jest.fn(),
+  navigate: jest.fn(),
+};
+jest.mock('../../src/services/authFacebook');
+jest.mock('@react-native-community/async-storage');
+jest.mock('react-native-flash-message');
+
+describe('Login', () => {
+  const mockStore = configureStore();
+  const store = mockStore({});
+  const apolloStore = createApolloStore(USER_LOGIN_MUTATION);
+
+  const component = (
+    <Wrapper apolloStore={apolloStore} reduxStore={store}>
+      <Login navigation={navigation} />
+    </Wrapper>
+  );
+
+  it('shows loading when press login button', async () => {
+    const { getByText, getByTestId } = render(component);
+
+    const btn = getByText('Entrar com facebook');
+
+    fireEvent.press(btn);
+
+    await waitForElement(() => getByTestId('loading'));
+  });
+
+  it('when login is successful', async () => {
```

Comment:
When you gonna describe a scenario like "when whatever" you use `describe`. On `it` you put the expected response. Ex:

describe => when login is sucessful
it => Call expected function or show "blabla" message to user

Reply:
It makes sense! I'll improve that. Thanks!